common/scaffolding/interactive-content-en.html

```diff
@@ -0,0 +1,387 @@
+
+<h2>The <code>object</code> element</h2>
+<h3>An embedded image</h3>
+<object data="img/input-image-sample.jpg" width="300" height="200"></object>
+<pre><code>&lt;object data="img/input-image-sample.jpg" width="300" height="200"&gt;&lt;/object&gt;</code></pre>
+
+<h3>An embedded HTML page</h3>
+<object data="https://www.canada.ca/en.html" width="500" height="200"></object>
+<pre><code>&lt;object type="text/html" src="https://www.canada.ca/en.html" width="500" height="200"&gt;&lt;/object&gt;</code></pre>
+
+<h3>An embedded video</h3>
+<object data="https://wet-boew.github.io/wet-boew-attachments/videos/video1-en.webm" width="400" height="300"></object>
+<pre><code>&lt;object data="https://wet-boew.github.io/wet-boew-attachments/videos/video1-en.webm" width="400" height="300"&gt;&lt;/object&gt;</code></pre>
+
+<h2>The <code>video</code> element</h2>
+<video controls>
+	<source src="https://wet-boew.github.io/wet-boew-attachments/videos/video1-en.webm" type="video/webm">
+	Your browser does not support the video tag.
+</video>
+<pre>
+<code>&lt;video controls&gt;
+&lt;source src="https://wet-boew.github.io/wet-boew-attachments/videos/video1-en.webm" type="video/webm"&gt;
+Your browser does not support the video tag.
+&lt;/video&gt;</code>
+</pre>
+
+<h2>The <code>audio</code> element</h2>
+<audio controls>
+	<source src="https://www.archive.org/download/RideOfTheValkyries/ride_of_the_valkyries_2.mp3" type="audio/mp3"/>
+	<source src="https://www.archive.org/download/RideOfTheValkyries/ride_of_the_valkyries_2.ogg" type="audio/ogg"/>
+	Your browser does not support the audio tag.
+</audio>
+<pre>
+<code>&lt;audio controls&gt;
+&lt;source src="https://www.archive.org/download/RideOfTheValkyries/ride_of_the_valkyries_2.mp3" type="audio/mp3"/&gt;
+&lt;source src="https://www.archive.org/download/RideOfTheValkyries/ride_of_the_valkyries_2.ogg" type="audio/ogg"/&gt;
+Your browser does not support the audio tag.
+&lt;/audio&gt;</code>
+</pre>
+
+<h2>The <code>track</code> element</h2>
+<video controls>
+	<source src="https://wet-boew.github.io/wet-boew-attachments/videos/video2-en.webm" type="video/webm">
+	<source src="https://wet-boew.github.io/wet-boew-attachments/videos/video2-en.mp4" type="video/mp4">
+	<track src="demo/video2-captions-en.xml" kind="captions" data-type="application/ttml+xml" srclang="en" label="English">
+	Your browser does not support the video tag.
+</video>
+<pre>
+<code>&lt;video controls&gt;
+&lt;source src="https://wet-boew.github.io/wet-boew-attachments/videos/video2-en.webm" type="video/webm"&gt;
+&lt;source src="https://wet-boew.github.io/wet-boew-attachments/videos/video2-en.mp4" type="video/mp4"&gt;
+&lt;track src="demo/video2-captions-en.xml" kind="captions" data-type="application/ttml+xml" srclang="en" label="English"&gt;
+Your browser does not support the video tag.
+&lt;/video&gt;</code>
+</pre>
+
+<h2>The <code>map</code> and <code>area</code> elements</h2>
+
```

Comment:
Need an example of the image map. It make sense to group those 2 elements together

Reply:
Fixed in commit "Addressed changes".